source/opt/module.cpp

```diff
@@ -137,7 +137,7 @@ void Module::ToBinary(std::vector<uint32_t>* binary, bool skip_nop) const {
   binary->push_back(header_.magic_number);
   binary->push_back(header_.version);
   // TODO(antiagainst): should we change the generator number?
```

Comment:
And the answer to this question is YES! 

A number incrementing by release would be awesome :)

Reply:
It's not clear that's the case.

For example, this would obscure the fact that every HLSL compilation through DXC would show up as the SPIR-V optimizer, because DXC post-processes initially generated code with a significant flow through the optimizer.